static/app/components/replays/virtualizedGrid/bodyCell.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+import TimestampButton from 'sentry/views/replays/detail/timestampButton';
+
+const cellBackground = p => {
+  if (p.isSelected) {
+    return `background-color: ${p.theme.textColor};`;
+  }
+  if (p.hasOccurred === undefined && !p.isStatusError) {
+    const color = p.isHovered ? p.theme.hover : 'inherit';
+    return `background-color: ${color};`;
+  }
+  const color = p.isStatusError ? p.theme.alert.error.backgroundLight : 'inherit';
+  return `background-color: ${color};`;
+};
+
+const cellColor = p => {
```

Comment:
i think there's a way to get a type for this but if not you could do `CellProps & {theme: Theme}`

Reply:
We may need to start looking into avoiding dynamic styles altogether as they are not very performant.

Reply:
If we aim to remove them altogether we should think about replacing `styled`.
i haven't tested yet, but it could be awkward to like half-use it and then our only fallback right now is the `style` prop/attr. 

as far as perf concerns go, idk how this stacks up to other wins, maybe if we claim something bigger we can keep on keeping on with this.